source/includes/reference/_profile_webhooks.md

```diff
@@ -0,0 +1,207 @@
+
+> Example Response:
+
+```json
+{
+    "id": "72195556-e5cb-495e-a010-b37a4f2a3043", 
+    "name": "Webhook Subscription #1",
+    "delivery": {
+        "version": "2.0.0",
+        "url": "https://your.webhook.url/12345" 
+    },
+    "trigger_on": "transfers#state-change", 
+    "scope": {
+        "domain": "profile", 
+        "id": "<profile ID>"
+    },
+    "created_by": {
+        "type": "user",
+        "id": "<your user ID>",
+    },
+    "created_at": "2008-09-15T15:53:00+00:00Z",
+}
+```
+
+### Request
+
+**`POST https://api.transferwise.com/v3/profiles/{profileId}/subscriptions`**
+
+`profileId` - ID of the profile you are subscribing to. 
+
+All fields listed below are required for creating a webhook subscription.
+
+Field                     | Description                                                             | Format
+---------                 | -------                                                                 | -----------
+name                      | A custom name for your webhook to ease with identification              | Text
+trigger_on                | `transfers#state-change`, `transfers#active-cases` or `balances#credit` | Text
+delivery.version          | The event representation semantic                                       | Text
+delivery.url              | Required. The URL where your server will be listening for events.       | Text
+
+
+### Response
+
+Field                     | Description                                                             | Format
+---------                 | -------                                                                 | -----------
+id                        | UUID that uniquely identifies the subscription                          | Text
+name                      | A custom name for your webhook to ease with identification              | Text
+trigger_on                | `transfers#state-change`, `transfers#active-cases` or `balances#credit` | Text
+delivery.version          | The event representation semantic                                       | Text
+delivery.url              | Required. The URL where your server will be listening for events.       | Text
+scope.domain              | Scope of this subscription, always "profile" in this case               | Text
+scope.id                  | Profile ID used to create this subscription                             | Text
+created\_by.type          | Creator type. Always user in this case                                  | Text
+created\_by.id            | User id of the creator                                                  | Text
+created\_at               | Timestamp of when the subscription was created                          | Text
+
+## Delete
+
+Deletes a subscription by its identifier.
+
+> Example Request:
+
+```shell
+curl -X DELETE "https://api.transferwise.com/v3/profiles/{profileId}/subscriptions/{id}" \
+  -H "Authorization: Bearer <your user token>"
+```
+
+> Example Response:
+
+```json
+{
```

Comment:
So what was the outcome of the test do we return 204 with empty body?

Reply:
Yes, we return 204 with empty body